build/packaging/linux/bin/kscnd

```diff
@@ -308,8 +308,12 @@ start_node() {
         fi
     fi
 
-    if [ ! -z $SC_DEFAULTGASLIMIT ]; then
-        OPTIONS="$OPTIONS --scdefaultgaslimit $SC_DEFAULTGASLIMIT"
+    if [ ! -z $SC_PARENT_OPERATOR_DEFAULTGASLIMIT ]; then
```

Comment:
`SC_PARENT_OPERATOR_DEFAULT_GAS_LIMIT` seems better.

Reply:
`gaslimit` should be single word. So I changed to `SC_PARENT_OPERATOR_GASLIMIT`.